Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -13,6 +13,26 @@ public class SpherePointer : BaseControllerPointer, IMixedRealityNearPointer
 
         public override SceneQueryType SceneQueryType { get { return raycastMode; } set { raycastMode = value; } }
 
+        [SerializeField]
+        [Min(0.0f)]
+        [Tooltip("Additional distance between SphereCastRadius and NearObjectRadius")]
+        private float nearObjectMargin = 0.2f;
+        /// <summary>
+        /// Additional distance between <see cref="BaseControllerPointer.SphereCastRadius"/> and <see cref="NearObjectRadius"/>.
+        /// </summary>
+        /// <remarks>
+        /// This creates a dead zone in which far interaction is disabled before objects become grabbable.
+        /// </remarks>
+        public float NearObjectMargin => nearObjectMargin;
```

Comment:
Does this (and `NearObjectRadius`) need to be public? They seem like values only used internally in this script.
Making things public for tests feels strange to me.

Reply:
I could thing of an example: what if you wanted to show an object *just out of reach*? So, at the point where you still have a hand ray but if you move just a bit closer the ray disappears. How would a consumer determine that distance without this var?

Reply:
I like to have at least a public getter for serializable fields. It seems wrong to be able to edit something in the inspector but having no way to at least _read_ that value at runtime.

Reply:
It's an aspect of Unity i'm really not fond of: this fact that you have more access to settings in the editor by default than through the API. At least that's how our access policy with private fields plays out in practice.

Reply:
Works for me! Thanks for the additional context.